tailwind.config.js

```diff
@@ -86,7 +87,7 @@ module.exports = {
         "text-gray": "#5c5c5c",
         "hr-red-bar": "#AE3B43",
         "canada-ca-link-colour": "#7834BC",
-        "circle-color": "#083C6C",
+        "circle-color": "#062F56",
```

Comment:
Was there a reason to change this one too? The Figma designs still show `#083C6C` for the background colour for the phase banner and the CallToAction component.

Reply:
I made it slightly darker because the original one will require a lighter button color to meet the required contrast ratio, but will not pass the contrast check again because not enough contrast with the white button text.